cmd/util/ledger/util/payload_file.go

```diff
@@ -239,8 +238,8 @@ func writeSelectedPayloads(logger zerolog.Logger, w io.Writer, payloads []*ledge
 	return includedPayloadCount, nil
 }
 
-func includePayloadByAddresses(payload *ledger.Payload, addresses []common.Address) (bool, error) {
-	if len(addresses) == 0 {
+func includePayloadByOwners(payload *ledger.Payload, owners map[string]struct{}) (bool, error) {
```

Comment:
I think it might be better to just always include the system registers, because it doesn't really make sense not to. They are small, so if you want to do further migrations with the resulting data they wont affect much and you basically need them to properly use the resulting state in transactions and scripts.

Reply:
Yeah, I was thinking of doing that, thanks for seconding it. I was not aware of the empty registers and given we only test/develop with filtered state locally, I accidentally broke the migration. Will make the appropriate change

Reply:
Added in [a82f463](https://github.com/onflow/flow-go/pull/5939/commits/a82f4630e9999c429ac5ccca9caa931b3501cb99)